plotpal/volumes.py

```diff
@@ -859,7 +881,7 @@ def plot_spheres(self, start_fig=1, **kwargs):
         with self.my_sync:
             tasks = []
             for k, sp in self.spheres:
-                for task in sp.equator + sp.left_meridian + sp.right_meridian + [sp.outer_shell,]:
+                for task in sp.equator + sp.left_meridian + sp.right_meridian + [sp.inner_shell,sp.outer_shell]:
```

Comment:
probably we should make this a bit better (i did it bad in the first places)

Add a line above that is tasks = sp.equator + sp.left_meridian + sp.right_meridian + [sp.outer_shell]
then if inner raidius > 0 do tasks += [sp.outer_shell]
then do for task in tasks: